src/persistence/profile.cpp

```diff
@@ -324,7 +324,7 @@ void Profile::saveToxSave(QByteArray data)
 bool Profile::exists(QString name)
 {
     QString path = Settings::getInstance().getSettingsDirPath() + name;
-    return QFile::exists(path+".tox") && QFile::exists(path+".ini");
+    return QFile::exists(path+".tox") && QFile::exists(path+".ini") && QFile::exists(path+".qtox_history");
```

Comment:
It isn't right, if the user unchecked "Keep chat history", then there isn"t a ".qtox_history" associated to their profile.